htdp-lib/2htdp/private/check-aux.rkt

```diff
@@ -53,8 +60,8 @@
 ;; MouseEvent% -> [List Nat Nat MouseEventType]
 ;; turn a mouse event into its pieces 
 (define (mouse-event->parts e)
-  (define x (- (send e get-x) INSET))
-  (define y (- (send e get-y) INSET))
+  (define x (- (send e get-x) (universe-inset)))
```

Comment:
Use `(define INSET (universe-inset))` and then leave these lines alone.  (It's not like INSET will change during the course of the `big-bang` program.  (Yes this is a "contract" that we don't enforce right now.))